src/components/PlioQuestion.vue

```diff
@@ -99,14 +106,19 @@ export default {
       text: "",
       selectedOption: null,
       isAnswerCorrect: false,
-      isAnswerSubmitted: false
+      isAnswerSubmitted: false,
+      showButtonLoading: false
     };
   },
 
   computed: {
-    // Submit button disabled if no option selected
+    // Submit button disabled if no option selected or screen is loading
     isDisabled() {
-      return (this.selectedOption == null || this.isAnswerSubmitted == true);
+      return (
+        this.selectedOption == null 
+        || this.isAnswerSubmitted == true 
+        || this.showButtonLoading == true
+      );
```

Comment:
Disable all the other buttons when there's a loading spinner running on the question pop-up